docs/transactions.md

```diff
@@ -685,8 +685,8 @@ Approves the proposed root certificate.
 The certificate is not active until sufficient number of Trustees approve it.
 
 - Parameters:
-  - subject: `string`  - proposed certificates's `Subject`
-  - subject_key_id: `string`  - proposed certificates's `Subject Key Id`
+  - subject: `string`  - proposed certificates's `Subject`in base64 format
+  - subject_key_id: `string`  - proposed certificates's `Subject Key Id` in hex string format
```

Comment:
Please provide more details description what does hex string means here in in other places where we mention `subject_key_id`.

Reply:
Done! ✅